### DISCUSSION TITLE
How do I ignore validation for a single field

### DISCUSSION THREAD

**Discussion by BrutalSimplicity** [pydantic/pydantic#4070] [Question] [answered]
I would like to ignore validation only for certain fields. I'd still like to be able to assign a value to and have the type system believe it is the value I defined. Don't ask why? 😄  How can I do this?

**Comment by BrutalSimplicity** [comment, marked as answer]
I found what will work in my case. The `arbitrary_types_allowed` is a less explicit way to achieve this as long as you set the field to a non-pydantic type (i.e. doesn't inherit from `BaseModel`)

https://pydantic-docs.helpmanual.io/usage/types/#arbitrary-types-allowed

**Comment by nythrox** [comment]
With pydantic v2, seems like you can use [SkipValidation](https://docs.pydantic.dev/2.0/api/functional_validators/#pydantic.functional_validators.SkipValidation) now
